### PR TITLE
[14.0][MIG] stock_inventory_analytic

### PR DESCRIPTION
**Pull request by rousseldenis**
- [x] Improved tests

### CODE REVIEW

**Comment by rousseldenis** [comment]
@OCA/accounting-maintainers

**Comment by kittiu** [comment]
Reviewed and tested, LGTM.

**Review by kittiu — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Code review, looks good :+1:

File: stock_inventory_analytic/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Stock Inventory Analytic",
+    "summary": """
+        Stock Inventory Analytic """,
+    "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (AaronHForgeFlow) [inline stock_inventory_analytic/__manifest__.py]: Am I too picky if I ask you to change Eficent by ForgeFlow?

File: stock_inventory_analytic/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
```
Comment (AaronHForgeFlow) [inline stock_inventory_analytic/__manifest__.py]:
```suggestion
# Copyright 2019 ForgeFlow S.L.
```
Here as well if possible :)

Comment (rousseldenis) [reply stock_inventory_analytic/__manifest__.py]: Of course ! I do not want to be the source of confusion :-)

**Comment by rousseldenis** [comment]
@AaronHForgeFlow done !

**Pull request merged**